source/common/quic/quic_filter_manager_connection_impl.cc

```diff
@@ -176,25 +176,15 @@ void QuicFilterManagerConnectionImpl::onConnectionCloseEvent(
     return;
   }
   switch (version.transport_version) {
-  case quic::QUIC_VERSION_43:
```

Comment:
Since these counters are no longer used. I think they can be removed from docs/root/configuration/http/http_conn_man/stats.rst and source/common/http/http3/codec_stats.h